### PR TITLE
Feature/create consultations

### PR DESCRIPTION
**Pull request by Rabid-Dan**
Fulfils https://trello.com/c/lcztAx2Q/63-create-a-consultation

Adds a migration for the consultation model as well as some basic controller actions, validations on the model and some basic views.
The views allow a user to create a new consultation and view the details of that consultation.

This PR also contains some extra test setup that was missed in the template such as adding Faker and setting up capybara to run feature specs with headless chrome.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -31,3 +31,8 @@
 
 en:
   hello: "Hello world"
+  consultation:
+    name: "Name"
+    type: "Consultation type"
+    create:
+      failure: "Consultation failed to create"
```
Comment (mermop) [inline config/locales/en.yml]:
```suggestion
      failure: "Failed to create consultation"
```

File: spec/models/consultation_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  it { is_expected.to have_attribute :name }
+  it { is_expected.to have_attribute :consultation_type }
+
+  it "is invalid without a name" do
+    subject.name = ""
+    expect(subject).not_to be_valid
+  end
+
+  it "is invalid without a consultation_type" do
```
Comment (mermop) [inline spec/models/consultation_spec.rb]: Might also be worth testing an invalid `consultation_type`

File: config/locales/en.yml (hunk 1 above)
Comment (Rabid-Dan) [reply config/locales/en.yml]: 👏 I thought the original felt a bit clunky, I'm gonna change it to this.

File: spec/models/consultation_spec.rb [hunk 3]
```diff
@@ -15,4 +15,8 @@
     subject.consultation_type = ""
     expect(subject).not_to be_valid
   end
+
+  it "is invalid with a consultation_type not included in the enum" do
+    expect { subject.consultation_type = "hotdog" }.to raise_error(ArgumentError)
```
Comment (mermop) [inline spec/models/consultation_spec.rb]: ✨

**Comment by mermop** [comment]
Travis is upset about code coverage being too low, oh no!

**Pull request merged**